src/compat/compat.h

```diff
@@ -37,14 +36,13 @@
 	(is_supported_pg_version_13(version) || is_supported_pg_version_14(version) ||                 \
 	 is_supported_pg_version_15(version) || is_supported_pg_version_16(version))
 
-#define PG12 is_supported_pg_version_12(PG_VERSION_NUM)
 #define PG13 is_supported_pg_version_13(PG_VERSION_NUM)
 #define PG14 is_supported_pg_version_14(PG_VERSION_NUM)
 #define PG15 is_supported_pg_version_15(PG_VERSION_NUM)
 #define PG16 is_supported_pg_version_16(PG_VERSION_NUM)
 
-#define PG13_LT (PG_VERSION_NUM < 130000)
-#define PG13_GE (PG_VERSION_NUM >= 130000)
+#define PG13_LT (PG_VERSION_NUM < 130002)
+#define PG13_GE (PG_VERSION_NUM >= 130002)
```

Comment:
You can't change those macros because they're used to specify the boundaries of a major Postgres version... for this 13.2 stuff we changed the is_supported_pg_version_13 macro.

Reply:
Okay. I have updated the PR to remove this change.